libs/block-mesh-manager/migrations/20240822174812_create_analytics_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE analytics
+(
+    user_id          UUID        NOT NULL REFERENCES users (id),
+    depin_aggregator TEXT        NOT NULL,
+    device_type      TEXT        NOT NULL,
+    created_at       TIMESTAMPTZ NOT NULL,
+    updated_at       TIMESTAMPTZ NOT NULL,
+    PRIMARY KEY (user_id, depin_aggregator)
```

Comment:
i prefer simple primary key and adding a composite unique index.
since if you want to refer to this table in the future, much simpler with a simple primary key vs. a composite one